pandas/tests/io/parser/test_parse_dates.py

```diff
@@ -2237,3 +2237,17 @@ def test_parse_dates_arrow_engine(all_parsers):
         }
     )
     tm.assert_frame_equal(result, expected)
+
+
+def test_from_csv_with_mixed_offsets():
```

Comment:
does it work to use the
```
parser = all_parsers
```
and then
```
parser.read_csv
```
pattern from the other tests in this file?

Reply:
Yes, it works very well, except `pyarrow`. I added `@xfail_pyarrow`.